apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx

```diff
@@ -0,0 +1,43 @@
+import { translations } from '@/_contents/translations';
+import RelatedLinks from '@/components/atoms/RelatedLinks/RelatedLinks';
+import StartInfo from '@/components/organisms/StartInfo/StartInfo';
+import { getWebinar, getWebinars } from '@/lib/api';
+
+type Params = {
+  webinarSlug: string;
+};
+
+export async function generateStaticParams() {
+  const webinars = await getWebinars();
+  return [...webinars].map(({ slug }) => ({
+    webinarSlug: slug,
+  }));
+}
+
+const Page = async ({ params }: { params: Params }) => {
+  const webinar = await getWebinar(params?.webinarSlug);
+  const { webinar: webinarLabels } = translations;
+
+  return (
+    <>
+      {webinar.startInfo && (
+        <StartInfo
```

Comment:
The background color of this section is not #FAFAFA, but #F2F2F2 and the title is too close to the cards.
![image](https://github.com/pagopa/developer-portal/assets/39835990/c38e5d06-eb2f-444d-8170-c0050d203bb5)